service.go

```diff
@@ -144,6 +149,10 @@ func (s *service) Push(ctx context.Context, req *pb.PushRequest) (*pb.PushReply,
 		if err = rec.Verify(logpk); err != nil {
 			return nil, err
 		}
+
+		if err = s.handleLogUpdate(ctx, req.ThreadID.ID, req.LogID.ID, rec); err == nil {
```

Comment:
Attempt to process this record as a log update, meaning that the record body will have log info.

Reply:
(This happens when a peer that has added a follower writes the new address to their own log. The other peers need to update there addresses as well.)